src/uu/ls/src/ls.rs

```diff
@@ -2482,13 +2482,13 @@ fn display_item_long(
             }
         };
 
-        let dfn = display_file_name(item, config, None, String::new(), out).contents;
+        let display_file_content = display_file_name(item, config, None, String::new(), out).contents;
```

Comment:
no need to make it that long, please call it
displayed_file

and it isn't the content of the file. it is just the filename

Reply:
Sure, just changed it.

I got a bit confused with ".contents" property.

Reply:
no worries :)